prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -304,13 +304,16 @@ static int proc_conn_resp(struct poll_fd_mgr *poll_mgr,
 	cm_entry->fid = poll_info->fid;
 	memcpy(cm_entry->data, poll_info->cm_data, poll_info->cm_data_sz);
 
+	ret = tcpx_ep_msg_xfer_enable(ep);
+	if (ret)
+		goto err;
+
```

Comment:
Can you describe the problem that this change fixes in more detail in the commit message?

Reply:
In case of failure of  call - tcpx_ep_msg_xfer_enable, the current code sends an error to eq after FI_CONNECTED event is sent to app. With this modification it would send error first.  I think this change is desirable since app gets the error sooner than later. If there are other problems that this is solving I would be happy to know.